docs/explanation/agent-internals.mdx

```diff
@@ -69,11 +70,21 @@ In a backend application, we suggest using the [Protobuf compiler](https://proto
 
 Once connected, an agent will receive chunks of audio as [`AgentRequest.TrackData` protobuf message](../api/reference#protobufs).
 
-### Step 4. Disconnecting from the Room
+### Step 4. Sending Audio
+
+All of further steps are done via the websocket connection created in [Step 2](#step-2-connecting-to-the-room).
+
+To send audio to an agent, you need to create an audio track first.
+This is done by sending an [`AgentRequest.CreateTrack` protobuf message](../api/reference#protobufs).
```

Comment:
Ideally, we would have docs generated from the protobuf files, instead of linking to index of Api Reference page